kit/src/main/resources/com/oracle/javafx/scenebuilder/kit/i18n/SceneBuilderKit.properties

```diff
@@ -430,10 +430,19 @@ error.file.create.title = File creation failed
 # {0} is a file path
 error.file.create.message = File {0} cannot be created
 error.file.open.message = File {0} cannot be opened
-error.file.open.title = File opening failed
 error.write.details = Please check your file system.
-error.file.reveal.title = File locating failed
-error.file.reveal.message = File {0} cannot be found
+
+# Alert file open error
+alert.error.file.open.title = File opening failed
```

Comment:
do these need translation?

Reply:
I think eventually yes, but at this very moment I am not actually capable to properly translate into japanese or chinese. Any good idea to achieve this automatically? One idea was to ask @ktgw0316 for the Japanese translation and possibly @weg2022 for the Chinese one.

Reply:
I've files issue #734 for this.